### PR TITLE
adds missing cron output binding

### PR DESCRIPTION
**Pull request by mchmarny**
# Description

Cron scheduler input binding provides an option to stop schedule execution through output binding. This adds the missing output binding registration in main. 

## Checklist

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [x] Extended the documentation

### CODE REVIEW

**Comment by amanbha** [comment]
@mchmarny Please create an issue for it as well.

File: cmd/daprd/main.go [hunk 1]
```diff
@@ -350,6 +351,9 @@ func main() {
 			bindings_loader.NewOutput("azure.eventgrid", func() bindings.OutputBinding {
 				return eventgrid.NewAzureEventGrid(logContrib)
 			}),
+			bindings_loader.NewOutput("cron", func() bindings.OutputBinding {
```
Comment (amanbha) [inline cmd/daprd/main.go:354]: does this needs to be registered as input as well as cron is implementing both Read and Invoke.

Comment (mchmarny) [reply cmd/daprd/main.go:354]: Input is already registered in the v0.9 release. This is just adding the output binding

**Comment by mchmarny** [comment]
Issue #1804

File: cmd/daprd/main.go (hunk 1 above)
Comment (amanbha) [reply cmd/daprd/main.go:354]: got it, LGTM overall, looks like there are some conflicts, could you please resolve them.

**Pull request merged**